Source/WebCore/layout/formattingContexts/inline/InlineContentCache.h

```diff
@@ -61,9 +64,12 @@ class InlineContentCache {
         bool hasInlineBoxes() const { return !!inlineBoxCount(); }
         size_t inlineBoxCount() const { return m_contentAttributes.inlineBoxCount; }
 
+        const ElementBoundaryTextSpacings& elementBoundaryTextSpacings() const { return m_elementBoundaryTextSpacings; }
+
     private:
         ContentAttributes m_contentAttributes;
         InlineItemList m_inlineItemList;
+        ElementBoundaryTextSpacings m_elementBoundaryTextSpacings;
```

Comment:
This looks to me like a side structure to InlineItems, so should probably go _next_ to InlineItems.